packages/react-next/src/components/Slider/Slider.base.tsx

```diff
@@ -28,274 +23,82 @@ const useComponentRef = (props: ISliderProps, thumb: React.RefObject<HTMLSpanEle
     [value],
   );
 };
+const SliderLabel = (props: {
+  className?: string;
+  label?: string;
+  ariaLabel?: string;
```

Comment:
as discussed in https://hackmd.io/k2eJKI_JRs2Gew9dbrz00A?both, can you pls use dashed aria props? See https://reactjs.org/docs/dom-elements.html. camelCased can cause inconsistencies when spreading unhandled props to the root element.